src/app/containers/ArticleTimestamp/helpers.test.js

```diff
@@ -112,21 +112,21 @@ describe('ArticleTimestamp helper functions', () => {
     });
 
     it(`should return date and time with timezone format when firstPublished is today`, () => {
-      const firstPublished = timestampGenerator({ hours: 4 });
+      const firstPublished = timestampGenerator({ days: 0 });
```

Comment:
This is a little hard to understand. What does `days: 0` actually generate?

Reply:
@jamesdonoh `timestampGenerator({ days: 0 })` generates timestamp for the current day ie. anytime the test runs it's equated to be the time at that moment

Reply:
Thanks for adding more context to the PR - was helpful.

Looking at the implementation of `timestampGenerator` I think `{ days: 0 }` (or `{ minutes: 0 }` or anything else) is the same as passing no argument at all to the function. Which means in turn that L115 is identical to:

```javascript
const firstPublished = Date.now();
```

Reply:
My worry is that it changes the original meaning of the tests, they will pass but they will not be testing exactly what we think they're testing. I think we need a way of simulating these scenarios that are not dependent on the current time at all, e.g. by looking at the feasibility of stubbing Date.now while these tests are run.

Reply:
thanks @jamesdonoh yes that's correct, would change to use `Date.now()`